hud-data/ultrahud-tf2.json

```diff
@@ -2,11 +2,13 @@
   "name": "ULTRAHUD-TF2",
   "author": "The 2fort Cremator",
   "contributors": ["raysfire", "Village Green Preserver", "Tomma", "techno"],
+  "parent": "rayshud",
+  "distinguishedFromParent": true,
   "releaseDate": "2024-02-24",
-  "resources": ["divine-intervention"],
+  "resources": ["item-update"],
```

Comment:
I see that you also have `ultrahud-logo.webp` in your `hud-resources` folder, is this something you'd want to include under resources as well?

Reply:
No. Leave it as it is.